getgauge/static_loader.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from os import path
+from redbaron import RedBaron
+from baron.utils import BaronError
+
+from getgauge.util import *
+from getgauge.registry import registry
+
+
+def load_file(content, file_name):
+    try:
+        red = RedBaron(content)
+        for func in red.find_all('def'):
+            for decorator in func.decorators:
+                steps = re.findall(r'[\'"](.*?)[\'"]', decorator.call.__str__())
```

Comment:
need to check that the decorator is a `step`. else it may match other decorators.